BootloaderCorePkg/BootloaderCorePkg.dec

```diff
@@ -38,6 +38,9 @@
   gPlatformModuleTokenSpaceGuid.PcdFlashSize              | 0x00000000 | UINT32 | 0x20000011
   gPlatformModuleTokenSpaceGuid.PcdFlashBlockSize         | 0x00001000 | UINT32 | 0x20000012
 
+  gPlatformModuleTokenSpaceGuid.PcdFSPT64Bit              | FALSE      | BOOLEAN | 0x20000013
+  gPlatformModuleTokenSpaceGuid.PcdFSPM64Bit              | FALSE      | BOOLEAN | 0x20000014
+  gPlatformModuleTokenSpaceGuid.PcdFSPS64Bit              | FALSE      | BOOLEAN | 0x20000015
```

Comment:
could we reduce the PCD numbers? e.g.: I don't think FSPM and FSPS are built using different arch.